### ISSUE TITLE
How can I use related fields in chart?

### ISSUE THREAD

**Issue by mixartemev** [LaravelDaily/laravel-charts#1]
For ex: I have posts with user_id. I want to show bars with user.name by their posts count.

**Comment by PovilasKorop** [comment]
@mixartemev currently it's not possible, we delayed it for future versions. It's pretty hard to code so flexibly for all relationships.

**Comment by PovilasKorop** [comment]
@mixartemev ok found a relatively easy way of doing what you need, released a new version:

https://github.com/LaravelDaily/laravel-charts/releases/tag/0.1.6
https://github.com/LaravelDaily/laravel-charts/commit/0236998684a0a37c0af96d9181ad88d415adb20a

**Comment by mixartemev** [comment]
@PovilasKorop thank you so much for reaction, but I can't see relation fields.
When it old style:
```
...
            'report_type'        => 'group_by_string',
            'group_by_field'     => 'region_id',
...
```
it works fine, but without relation names, only with ids from the same table
![joxi_screenshot_1560197313473](https://user-images.githubusercontent.com/5181924/59223482-b48d5680-8bd4-11e9-9dca-26cecd6b4bd7.png)

But if I change options to group_by_relationship:
```
...
            'report_type'        => 'group_by_relationship',
            'relationship_name'  => 'region',
            'group_by_field'     => 'name',
...
```
the chart is empty, without any errors
![joxi_screenshot_1560197183328](https://user-images.githubusercontent.com/5181924/59223659-22d21900-8bd5-11e9-895a-e56ae94466c1.png)

(ps: I've updated package to 0.1.6 version, of course)

**Comment by mixartemev** [comment]
I found out the reason: my table 'buildings' haven't field 'name' (only 'region' table has field 'name'), but it's checked in initial table instead related table.
fixed in #2
